src/apis/emscripten/utils.rs

```diff
@@ -92,29 +92,43 @@ pub unsafe fn copy_terminated_array_of_cstrs(
     0
 }
 
+#[repr(C)]
+pub struct GuestStat {
+    st_dev: u64,
+    st_ino: u32,
+    st_mode: u32,
+    st_nlink: u32,
+    st_uid: u32,
+    st_gid: u32,
+    st_rdev: u64,
+    st_size: u32,
+    #[cfg(not(target_os = "windows"))]
```

Comment:
The wasm representation doesn't change if the host is running on linux, darwin, or windows. It's invalid to change the behavior depending on the host environment.

Reply:
good feedback, I will default them! Thanks guys!